parsel/selector.py

```diff
@@ -189,7 +189,9 @@ def __init__(self, text=None, type=None, namespaces=None, root=None,
 
         if text is not None:
             if not isinstance(text, six.text_type):
-                raise TypeError("text argument should be of type %s" % six.text_type)
+                msg = "text argument should be of type %s, got %s" % (
+                           six.text_type, text.__class__)
```

Comment:
_Nitpick alert_ :warning:

Indentation seems weird to me in this line, it has 11 more spaces than the surrounding ones.
Personally, I'd put everything in the same line. It would be 100 chars, including the 16 spaces at the beginning, and there is already [a longer line](https://github.com/scrapy/parsel/blob/4be716dd2bcf6fbbb698a4d56a69e1c47d002068/parsel/selector.py#L84) in the file. But I do understand if that's not an option, I know I have a personal issue with the 79-char rule :smile:

Reply:
@kmike I’m in favor of keeping it in a separate line, but the indentation does seem weird.